cli/inspector.rs

```diff
@@ -87,12 +87,13 @@ impl InspectorServer {
 
 /// Inspector information that is sent from the isolate thread to the server
 /// thread when a new inspector is created.
+#[derive(Clone)]
 struct InspectorInfo {
   host: SocketAddr,
   uuid: Uuid,
   thread_name: Option<String>,
   new_websocket_tx: UnboundedSender<WebSocket>,
-  canary_rx: oneshot::Receiver<Never>,
+  canary_rx: Arc<Mutex<oneshot::Receiver<Never>>>,
```

Comment:
This is not right. If yo have multiple clones in different threads, the canary channel won't drop and therefore not work.